### PR TITLE
Add New York recommendations

### PR DESCRIPTION
**Pull request by robb1891**
This Pull Requests closes issue #67.
## Adding restaurants and activities
- Does anyone have a favorite food?
- Do you like indoor or outdoor activities?

### CODE REVIEW

File: robb1891/robb1891-NewYork.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## New York
+
+#### Restaurants
+
+#### Things to do
```
Comment (Redfire2001) [inline robb1891/robb1891-NewYork.md:7]: @robb1891  I love Asian and Indian food.  Are there any sports-related activities available? :basketball:

Comment (robb1891) [inline robb1891/robb1891-NewYork.md:7]: Yes. New York has plenty 💯

**Comment by doppeldimmykumont** [comment]
Best NY coffee? ☕ 

@doppelDimmykumont The best coffee I had was actually upstate in a town called Beacon.

**Comment by jdavis1998** [comment]
What is your favorite place to get a good lox n bagel?

@jdavis1998 Any NY bagel place is great!

**Pull request merged**